### PR TITLE
fix(docs): Update installation manual

### PR DESCRIPTION
**Pull request by simon-cj**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes https://github.com/kleveross/klever-model-registry/issues/126

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README_zh.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ $ make docker-build
 ```
 
 ### 部署
-请参考 [docs/installer/install_zh.md](docs/installer/installation_zh.md)
+请参考 [docs/installation/installation_zh.md](docs/installation/installation_zh.md)
```
Comment (FogDong) [inline README_zh.md:79]: Can we move all the zh doc to `/docs/docs_zh`?

Comment (simon-cj) [reply README_zh.md:79]: New a issue to resolve it when write docs in this sprint, this issue close now

**Comment by simon-cj** [comment]
@gaocegege 
reviewing? or need a lgtm

File: README_zh.md (hunk 1 above)
Comment (FogDong) [inline README_zh.md:79]: I mean we can remove this `docs/installation/installation_zh.md` to `docs/docs_zh/installation.md`, and remove `docs/installer/install.md` to `docs/installation.md`. I think we'll only have one install doc so this install dir is not necessary?

Comment (simon-cj) [reply README_zh.md]: oh, get it, comments are addressed, PTAL

**Pull request merged**